ios/Runner/AppDelegate.swift

```diff
@@ -97,6 +97,16 @@ import flutter_local_notifications
         }
         if validateDisplayPushNotification(userInfo: notification.request.content.userInfo) {
             completionHandler([.alert, .badge, .sound])
+        } else if UIApplication.shared.applicationState == .active {
+            let viewController = window?.rootViewController as! FlutterViewController
+            let fcmMethodChannel = FlutterMethodChannel(
+                name: CoreUtils.FCM_METHOD_CHANNEL_NAME,
+                binaryMessenger: viewController.binaryMessenger)
+            fcmMethodChannel.invokeMethod(
+                CoreUtils.FCM_ON_MESSAGE_METHOD_NAME,
+                arguments: notification.request.content.userInfo)
```

Comment:
We should initial `MethodChannel` in `didFinishLaunchingWithOptions `

Reply:
Done